common/libs/units/units.h

```diff
@@ -132,6 +152,10 @@ template <class Q, class ValTy> class ArithScalar : public Scalar<Q, ValTy> {
 //  - cm h2o (centimeters of water)
 //
 // Native unit (implementation detail): kPa
+//
+// Note that multiplying and dividing pressure by a float makes little sense,
+// as they are not an absolute value.
+// Division may be used to define an average.
```

Comment:
Suggest

```Note that you can't multiply/divide a Pressure by a scalar; Pressure is an ArithScalar and not a ModuleScalar.  We did this because pressure is a relative quantity, and so multiplying/dividing by a scalar doesn't make sense in most contexts.  This does mean a little more work if you want to average N Pressure values.```

Reply:
Actually, I think I misunderstood your previous review... I made it so all the ArithScalar classes have a multiplication and division, including Pressure.
Hence this version of the comment.
Should I go back to both templates? (it is easy enough)

Reply:
Oh, I clearly didn't realize that you'd merged it into one template.  That's fine with me.

I don't think the comment is necessary at all, then.  Like, presumably whoever is reviewing code will be aware that multiplying/dividing pressure by a float does or doesn't make any sense.

Reply:
Yes, I'll get rid of this, we can assume that anyone using pressure should know what they are doing.